<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>892</width>
    <height>500</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>地图生成器</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="MapArea" name="mapLabel">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>720</width>
      <height>480</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>740</x>
      <y>150</y>
      <width>51</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>图块数：</string>
    </property>
   </widget>
   <widget class="QLabel" name="patternNum">
    <property name="geometry">
     <rect>
      <x>810</x>
      <y>150</y>
      <width>62</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>741</x>
      <y>190</y>
      <width>51</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>对数：</string>
    </property>
   </widget>
   <widget class="QLabel" name="pairNum">
    <property name="geometry">
     <rect>
      <x>810</x>
      <y>190</y>
      <width>62</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>740</x>
      <y>320</y>
      <width>62</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>图案种数：</string>
    </property>
    <property name="buddy">
     <cstring>kindNum</cstring>
    </property>
   </widget>
   <widget class="QSpinBox" name="kindNum">
    <property name="geometry">
     <rect>
      <x>820</x>
      <y>320</y>
      <width>54</width>
      <height>23</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="fileNameInput">
    <property name="geometry">
     <rect>
      <x>740</x>
      <y>420</y>
      <width>141</width>
      <height>23</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="generateMap">
    <property name="geometry">
     <rect>
      <x>800</x>
      <y>450</y>
      <width>85</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>生成文件</string>
    </property>
   </widget>
   <widget class="QPushButton" name="clear">
    <property name="geometry">
     <rect>
      <x>740</x>
      <y>230</y>
      <width>85</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>清空</string>
    </property>
   </widget>
   <widget class="QPushButton" name="fill">
    <property name="geometry">
     <rect>
      <x>740</x>
      <y>270</y>
      <width>85</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>填充</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>740</x>
      <y>390</y>
      <width>91</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>地图文件名：</string>
    </property>
    <property name="buddy">
     <cstring>fileNameInput</cstring>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>740</x>
      <y>50</y>
      <width>62</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>横向块数</string>
    </property>
    <property name="buddy">
     <cstring>yMax</cstring>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>740</x>
      <y>90</y>
      <width>62</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>纵向块数</string>
    </property>
    <property name="buddy">
     <cstring>xMax</cstring>
    </property>
   </widget>
   <widget class="QSpinBox" name="yMax">
    <property name="geometry">
     <rect>
      <x>810</x>
      <y>40</y>
      <width>55</width>
      <height>27</height>
     </rect>
    </property>
    <property name="value">
     <number>10</number>
    </property>
   </widget>
   <widget class="QSpinBox" name="xMax">
    <property name="geometry">
     <rect>
      <x>810</x>
      <y>90</y>
      <width>55</width>
      <height>27</height>
     </rect>
    </property>
    <property name="value">
     <number>8</number>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>740</x>
      <y>350</y>
      <width>62</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>每种对数：</string>
    </property>
   </widget>
   <widget class="QLabel" name="numPerPair">
    <property name="geometry">
     <rect>
      <x>820</x>
      <y>360</y>
      <width>62</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>MapArea</class>
   <extends>QLabel</extends>
   <header location="global">maparea.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
